src/lab_sim/config/moveit/picknik_ur.srdf

```diff
@@ -52,6 +52,7 @@
     <disable_collisions link1="base_link_inertia" link2="shoulder_link" reason="Adjacent"/>
     <disable_collisions link1="forearm_link" link2="upper_arm_link" reason="Adjacent"/>
     <disable_collisions link1="forearm_link" link2="wrist_1_link" reason="Adjacent"/>
+    <disable_collisions link1="base_link_inertia" link2="upper_arm_link" reason="Adjacent"/>
```

Comment:
I don't know why these links end up in collision after planning a waypoint, but this is my workaround